browser/BrowserPerf.proto

```diff
@@ -30,17 +30,19 @@ service BrowserPerfService {
 }
 
 message BrowserPerfData {
-    int32 serviceId = 1;
+    // UUID
+    string uniqueId = 1;
```

Comment:
@Fine0830 Is it OK to generate this unique for client-js report every time? The backend will need this to save the whole report for the UI. Are we going to provide perf data and error log query from UI?